database/export.go

```diff
@@ -106,7 +105,7 @@ func ExportHeaders(cfg *config.AppConfig, log *zerolog.Logger) error {
 }
 
 func queryDatabaseTable(db *sqlx.DB, log *zerolog.Logger) (*sqlx.Rows, error) {
-	rows, err := db.Queryx(selectHeadersSql)
+	rows, err := db.Queryx(selectHeadersSQL)
```

Comment:
as example here u doing with capital so why in other places not ?

Reply:
Answered about postgres